### ISSUE TITLE
Allow certain scenarios to bypass git lfs server-side hook

### ISSUE THREAD

**Issue by FattyMieo** [git-lfs/git-lfs#3279]
Hi, I'm having difficulties trying to do some automation for my got repository on GitLab. 

I have a script that automatically merge the `develop` branch to `release` branch on a scheduled time. The problem I'm facing is that my automation is getting blocked by `pre-receive` hook which prevents the merge to be pushed to `release` branch. There are still development going on behind the scene so I don't want to forcefully unlock the files before running the automation.

Another problem we have is with merge requests on GitLab. Currently, we have a rule to only unlock the files after the changes are merged into `develop`. We have set up some programmers to require approval for their features to be merged into `develop` via a merge request. The problem is, after the approval of a merge request, only the person submitting the files can do the merge. The server is unable to do the operation since the files are locked. We prefer the files to remained locked by the programmer-in-charge before the merge succeeds.

Is there any ways to set up a command, or a whitelist in the config to allow a specific account to bypass the `pre-receive` hook on Git LFS? I have searched for days but still doesn't seem to find any resources in this topic.

**Comment by FattyMieo** [comment]
Alright, thank you for guiding! I will try my luck on the GitLab side.
